src/sentry/static/sentry/app/views/settings/project/projectCreateServiceHook.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import DocumentTitle from 'react-document-title';
+import {RouteComponentProps} from 'react-router/lib/Router';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import ServiceHookSettingsForm from 'app/views/settings/project/serviceHookSettingsForm';
+
+type Props = RouteComponentProps<{orgId: string; projectId: string}, {}>;
+
+function ProjectCreateServiceHook({params}: Props) {
+  const {orgId, projectId} = params;
+  return (
+    <DocumentTitle title={t('Create Service Hook - Sentry')}>
```

Comment:
I would create a separate const for the `Create Service Hook ` and concat Sentry, because I believe "Sentry" is not something we would translate